tools/deb/postinst.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+action="$1"
+oldversion="$2"
+
+umask 022
+
+if ! getent passwd istio >/dev/null; then
+    addgroup --system istio
+    adduser --system --group --home /var/lib/istio istio
+fi
+
+if [ ! -e /etc/istio ]; then
+   # Backward compat.
+   ln -s /var/lib/istio /etc/istio
```

Comment:
which code hardcodes /etc/istio ? should we fix that (before 0.2 if not now)

Reply:
add TODO

Reply:
added in the BUILD (which needs to be updated first)